app/src/main/res/layout/country_preview_layout.xml

```diff
@@ -1,17 +1,15 @@
 <?xml version="1.0" encoding="utf-8"?>
-<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
What was the need switching to LinearLayout?

Reply:
Reverted.